### DISCUSSION TITLE
Lan on Monterey

### DISCUSSION THREAD

**Discussion by Qerope** [Mattis-Schulte/hackintosh-catalina#11] [General]
Have you tried using lan connection on Monterey? I wanted to upgrade but I have read [here](https://github.com/willza3/macOS-strix-B450i) that lan does not work on b450i which is a quite similar board.

**Comment by Mattis-Schulte** [comment]
I just made some changes that should make it possible to use a LAN connection, although there are still some issues with it, at least it kinda works.
